lib/Compilation.js

```diff
@@ -4086,7 +4086,7 @@ This prevents using hashes of each other and should be avoided.`);
 			if (!isSourceEqual(this.assets[file], source)) {
 				this.errors.push(
 					new WebpackError(
-						`Conflict: Multiple assets emit different content to the same filename ${file}`
+						`Conflict: Multiple assets emit different content to the same filename ${file}. Source File: ${assetInfo}`
```

Comment:
You can't print an object that way, you might want to `JSON.stringify` it, or print it in a nicer way.

Anyway, please also make sure to test the code. e. g. here is a test case you can update: `test/configCases/emit-asset/different-source`

Reply:
My apologies.
For testing, I have run the `yarn jest ConfigTestCases` and from my understanding the testing has been done in `ConfigTestCases.tempate.js` file which grabs the files from `configCases` folder and tests them. But still didn't understand what should I update in `test/configCases/emit-asset/different-source` ? Please, let me know about it.